src/scenes/Header/HeaderSearch/HeaderSearch.tsx

```diff
@@ -0,0 +1,38 @@
+import { Input, InputAdornment, makeStyles } from '@material-ui/core';
+import { Search } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+
+const useStyles = makeStyles(() => ({
+  search: {
+    width: '100%',
+    maxWidth: '545px',
+    border: '1px solid #D1DADF',
+    color: 'textSecondary',
+    padding: '4px',
+    borderRadius: '6px',
+    '&:before': {
+      display: 'none',
+    },
+    '&:after': {
+      borderBottomColor: 'primary',
+    },
+  },
```

Comment:
You shouldn't need any of these other than `width` & `maxWidth` after change below.
Also `width: 100%` can be applied with `fullWidth` prop on component.
Also `maxWidth: 545` - you declare it as a number and leave off the `px`.

Also by convention the class used on the "root" element in the component is called `root`.
The classes are already scoped to the file so this would be `HeaderSearch-root-###`